README.md

```diff
@@ -40,6 +40,45 @@ api.add_resource("/widgets", WidgetListView, WidgetView)
 
 Documentation is available at https://flask-resty.readthedocs.io/.
 
+## Running locally
+
+Ideally, create a virtual env for local development, using your tool of choice, e.g.:
+
+```sh
+pyenv virtualenv 3.8.3 flask-resty
+pyenv local flask-resty
+```
+
+Install dependencies:
+
+```sh
+# optionally include [jwt] for the full test suite, but requires cryptography,
+# and it's dependent binary libraries
+pip install -e .[testing,docs]
+```
+
+### Running tests
+
+Tests can be run for your current enviroment by running pytest directly: `pytest`.
+
+Or run `tox` to run the fulle suite of lint checks and tests against flask-resty's supported
+environments.
+
```

Comment:
suggestion: Might be useful to document `tox -e lint` here as well.

FWIW, here's how marshmallow documents it: https://marshmallow.readthedocs.io/en/stable/contributing.html#running-tests